.travis.yml

```diff
@@ -58,7 +58,6 @@ deploy:
   script: bash hack/gen-swagger-doc/deploy.sh
```

Comment:
@rmohr do you think that I need to execute differently, after you containerized build?

Reply:
I took a look at it and I think it is not necessary, it just clone api-reference repo, copy changes, commit and push them back.
So please ignore that comment above.